packages/e2e/test/pg-boss/stake-pool-metrics.test.ts

```diff
@@ -0,0 +1,44 @@
+import { Pool } from 'pg';
+import { STAKE_POOL_METRICS_UPDATE } from '@cardano-sdk/projection-typeorm';
+import { getEnv } from '../../src';
+
+const selectRowsCount = async (db: Pool, table: string) => {
+  const query = `SELECT COUNT(*) FROM ${table}`;
+  const { rows } = await db.query(query);
+
+  return rows;
+};
+
+describe('stake-pool-metrics', () => {
+  const key = 'STAKE_POOL_CONNECTION_STRING';
+  const connectionString = getEnv([key])[key];
+  const db = new Pool({ connectionString });
+
+  it('populates current_pool_metrics table', async () => {
+    let jobState: string[] | undefined;
+
+    // Wait until all the jobs are completed
+    while (jobState?.filter((_) => _ === 'completed').length === 0) {
```

Comment:
Isn't this `undefined === 0` on 1st check?

Reply:
Nice catch!
567ee08cc should solve this